joplin/js/CreateContentModal/ChooseTopicStep.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import plus from '../../static/images/plus.png';
+import minus from '../../static/images/minus.png';
+import unselected from '../../static/images/unselected.png';
+import selected from '../../static/images/selected.png';
+
+const TopicItem = ({ topic, handleTopicSelect, activeTopic }) => (
+  <li
+    onClick={handleTopicSelect.bind(this, topic.id)}
+    className="ChooseTopicStep__topic-toggle"
+  >
+    <div
+      className={
+        activeTopic === topic.id
+          ? "ChooseTopicStep__topic--selected"
```

Comment:
Maybe this could be fancier with:
``` `ChooseTopicStep__topic--${activeTopic === topic.id ? "selected" : "unselected"}` ```

after writing this not sure which way is more readable though